packages/init/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@slicemachine/init",
-	"version": "1.1.10-alpha.1",
+	"version": "1.1.10-alpha.4",
```

Comment:
is it normal/okay that we change the version from `alpha.1` to `alpha.4`?

Reply:
yes :) it comes from the main, which means several versions have been deployed (ticket merged), so i need the latest versions to make sure it is incremented properly for deployment when this will be merged.
I should normally have the same version as the v0.4.0.